docs/automated-tests/troubleshooting/use-partial-run.md

```diff
@@ -1,47 +1,47 @@
 
 ![Sample web app](/img/automated-tests/troubleshooting/use-partial-run/1Scenario.png)
 
-We’ve already launched <a href="/automated-tests/recording/overview#start-a-recording-session" target="_blank">Test Studio’s recorder and recorded</a> the following test steps:
+We’ve already launched <a href="/automated-tests/recording/overview#start-a-recording-session" target="_blank">Test Studio’s recorder</a> and recorded the following test steps:
 
 ![Steps in test](/img/automated-tests/troubleshooting/use-partial-run/2RecordedTest.png)
 
 ## Execute the Test and Explore the Results
 
-Upon <a href="/automated-tests/test-execution/quick-execution" target="_blank">execution of this sample test</a>, we expect it to pass successfully. However, the __verification in step 2. fails__ and the failure message clearly states that the ___button_ state is not enabled__. The _Images_ section shows that the correct page is loaded, which brings no hint for identifying the cause of the failure.
+Upon <a href="/automated-tests/test-execution/quick-execution" target="_blank">execution</a> of this sample test, we expect it to pass successfully. However, the verification in step 2 fails, and the failure message states that the button's state is not `enabled`. The _Images_ section shows that the correct page is loaded, which gives us no hints for the cause of the failure.
 
 ![Failure in test](/img/automated-tests/troubleshooting/use-partial-run/3Failingtest.png)
 
-This is where the <a href="/automated-tests/test-execution/partial-test-execution#run-to-here" target="_blank">__Run to Here__ feature</a> comes into action. The partial test run can be triggered from the context menu of a selected step - right click on the failing step and choose __Run -> To Here__.
+The <a href="/automated-tests/test-execution/partial-test-execution#run-to-here" target="_blank">__Run to Here__ feature</a> allows you to troubleshoot similar scenarios. You can trigger the partial test run from the context menu of a selected step - right-click the failing step and choose __Run > To Here__.
 
 ![Failure in test](/img/automated-tests/troubleshooting/use-partial-run/4RuntoHere.png)
 
-Choosing this option launches a new instance of the selected (or if set, <a href="/automated-tests/test-execution/quick-run-browsers#preferred-browser" target="_blank">preferred</a>) browser, __execute the test to the selected step, leave the browser open and the application in its current state and attaches the recorder__. The test run fails again, but there is an active recorder session running for this browser instance. That enables the other partial run options in the step context menu - to run <a href="/automated-tests/test-execution/partial-test-execution#run-selected-steps" target="_blank">__Selected Steps__</a> or to continue the run <a href="/automated-tests/test-execution/partial-test-execution#run-from-here" target="_blank">__From Here__</a>.
```

Comment:
"(or if set, <a href="/automated-tests/test-execution/quick-run-browsers#preferred-browser" target="_blank">preferred</a>) browser" - Makes the long sentence even longer. You can skip the preferred browser - it is documented in other articles.